products/green_fast_track/models/staging/stg__dcp_beaches.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
+    'beaches' AS variable_type,
+    agency || '-' || name AS variable_id,
+    st_union(wkb_geometry) AS raw_geom,
+    NULL AS buffer
```

Comment:
Should we be null'ing out the buffer, in general? When I don't have buffer specified, I've just been setting `raw_geom` and `buffer` to the same value